sound/soc/intel/boards/sof_sdw.c

```diff
@@ -225,7 +225,7 @@ int sdw_startup(struct snd_pcm_substream *substream)
 	return sdw_startup_stream(substream);
```

Comment:
Code looks ok, but can you please update the commit such that you put two "Fixes: " lines separately above signed-off and just have a generic description of the change in commit body.

Reply:
thanks!